apps/studio/pages/new/[slug].tsx

```diff
@@ -203,16 +204,22 @@ const Wizard: NextPageWithLayout = () => {
   const { data: organizations, isSuccess: isOrganizationsSuccess } = useOrganizationsQuery()
   const currentOrg = organizations?.find((o: any) => o.slug === slug)
 
+  const { data: orgSubscription } = useOrgSubscriptionQuery({ orgSlug: slug })
+
+  const { data: allOverdueInvoices } = useOverdueInvoicesQuery({
+    enabled: !['team', 'enterprise'].includes(orgSubscription?.plan.id ?? ''),
```

Comment:
This is going to fire immediately as orgSubscription will be undefined while it's still loading

```suggestion
    enabled: orgSubscription && !['team', 'enterprise'].includes(orgSubscription.plan.id),
```

Reply:
oh right gotcha on this! will update for DeployNewReplicaPanel too